drivers/dai/intel/ssp/dai-params-intel-ipc4.h

```diff
@@ -324,6 +324,7 @@ struct dai_intel_ipc4_ssp_configuration_blob {
 } __packed;
 
 #define SSP_BLOB_VER_1_5 0xee000105
+#define SSP_BLOB_VER_3_0 0xee000300
 
 struct dai_intel_ipc4_ssp_configuration_blob_ver_1_5 {
```

Comment:
Define separate "struct dai_intel_ipc4_ssp_configuration_blob_ver_3_0" that uses the PTL version of "struct dai_intel_ipc4_ssp_config".